code/modules/customitems/item_defines.dm

```diff
@@ -1253,7 +1253,7 @@
 	if(prompting_change)
 		return
 	prompting_change = TRUE
-	var/plushie_color = input("Select a color", "[src]") as null|anything in plush_colors
+	var/plushie_color = tgui_input_list(usr, "Select a color", "[src]", plush_colors)
```

Comment:
We should pass in the user from `ui_action_click` into change_color, so we can use it here

Reply:
Contra wants you to pass `user` from the calling proc into this, so you can use `user` instead of `usr`

Reply:
Hope i do right